components/brave_rewards/resources/tip/components/app.tsx

```diff
@@ -63,9 +63,9 @@ export class App extends React.Component<Props, {}> {
           url={url}
           monthly={monthly}
           publisher={publisher}
-          mediaMetaData={mediaMetaData}
           monthlyDate={monthlyDate}
           amount={currentTipAmount}
+          mediaMetaData={mediaMetaData}
```

Comment:
nit: can this be moved back where it was so that change is removed?

Reply:
Fixed.